jax/experimental/jax2tf/jax2tf.py

```diff
@@ -1332,6 +1332,28 @@ def _svd(operand, full_matrices, compute_uv):
 
 tf_impl[lax_linalg.svd_p] = _svd
 
+def _eig(operand: TfVal, compute_left_eigenvectors: bool,
+         compute_right_eigenvectors: bool):
+  if compute_left_eigenvectors and compute_right_eigenvectors:
+    # TODO(bchetioui): didn't find a 100% reliable, easy and satisfying way to
```

Comment:
It is not clear to me what sorting is relevant here. Presumably it has something to do with what tf.linalg.eigvals and tf.linalg.eig do?

Reply:
Given the triplet `(w, vl, vr)` of eigenvalues, left eigenvectors, and right eigenvectors, we expect that `vl[i]` is a left eigenvector with eigenvalue `w[i]`, and the same on the right for `vr[i]`, so the order needs to be consistent across these three.

Reply:
I understand, I suggest adding the clarification for the sorting in the comment itself.